terrain_planner/src/terrain_ompl_rrt.cpp

```diff
@@ -124,37 +123,34 @@ void TerrainOmplRrt::setupProblem(const Eigen::Vector3d& start_pos, const Eigen:
 
   double radius;
   if (goal_radius < 0) {
-    radius =
-        problem_setup_->getStateSpace()->as<fw_planning::spaces::DubinsAirplaneStateSpace>()->getMinTurningRadius();
+    radius = goal_radius;
   } else {
     radius = goal_radius;
   }
```

Comment:
This can be collapsed, radius no longer affects the result.

Reply:
Good catch, I will fix this.